docs/markdown/snippets/mintro_test_protocol.md

```diff
@@ -0,0 +1,4 @@
+## Test introspection data includes the test protocol
+
+The output of ``meson introspect --tests`` also includes the protocol to be used
+to run each test.
```

Comment:
Please move this into the existing snippet for the introspection changes (introspect.md).